iptables/iptables.go

```diff
@@ -304,15 +304,21 @@ func Exists(table Table, chain string, rule ...string) bool {
 
 	// try -C
 	// if exit status is 0 then return true, the rule exists
-	if _, err := Raw(append([]string{
-		"-t", string(table), "-C", chain}, rule...)...); err == nil {
+	_, err := Raw(append([]string{"-t", string(table), "-C", chain}, rule...)...)
+	if err == nil {
 		return true
 	}
+	if !strings.Contains(err.Error(), "unknown option") {
```

Comment:
Thanks @chenchun. Given probing with `-C` will not give a clear result about unsupported option, I am keen to use the version check and always run a `-C` check on versions >= 1.4.11.